src/css/imports/playlist.less

```diff
@@ -1,45 +1,40 @@
 @import "vars";
 
-
 .jw-playlist-container {
+    left: -51%;
```

Comment:
It moves the playlist container over - it's something that's done in every skin, but this value is specifically for the core styling (i.e. seven)